shell/platform/android/platform_view_android_jni.cc

```diff
@@ -47,15 +47,6 @@ bool CheckException(JNIEnv* env) {
 static fml::jni::ScopedJavaGlobalRef<jclass>* g_flutter_callback_info_class =
     nullptr;
 
-// FlutterView.java, from original embedding API. Not used in 2nd iteration of
```

Comment:
This file has a bunch of deleted stuff, but it doesn't seem to be replaced by anything.  Am I missing something?

Reply:
These deleted lines were binding the `native` methods in `FlutterNativeView.java` and `FlutterView.java`. I removed the native methods from those classes in this patch and changed them to wrap a `FlutterJNI` instance and call through JNI with that instead. So there shouldn't be any code needed in the C++ layer to bind or call those deleted methods anymore. The `FlutterJNI` versions of these methods are still being wired together in `Register(env)`.